src/renderers/shaders/ShaderChunk/envmap_fragment.glsl.js

```diff
@@ -40,7 +40,7 @@ export default /* glsl */`
 
 	#elif defined( ENVMAP_TYPE_CUBE_UV )
 
-		vec4 envColor = textureCubeUV( envMap, reflectVec, 0.0 );
+		vec4 envColor = textureCubeUV( envMap, reflectVec, backgroundBlurriness );
```

Comment:
This shader chunk is used only by Basic, Lambert, Phong, Toon, and Gouraud, none of which support ENVMAP_TYPE_CUBE_UV.

This shader chunk was designed for reflections, not for texturing a background box.

Reply:
How do you think we should do it instead?

Reply:
@mrdoob I think this `elif` should be removed from this shader chunk, and the background code should have its own chunks.

Let's merge the feature set we want first, then address these issues.